ebur128/ebur128.c

```diff
@@ -1457,3 +1457,42 @@ int ebur128_prev_true_peak(ebur128_state* st,
                      st->d->prev_sample_peak[channel_number]);
   return EBUR128_SUCCESS;
 }
+
+void ebur128_reset(ebur128_state *st) {
```

Comment:
This same effect could be achieved by the application by simply calling destroy() and init(). I think it would be worthwhile to discuss the need for this function (and the extra code to maintain). I am not against it, just opening discussion.

Reply:
Yes that's what I first think of but you would free and reallocate quite a few (other) things which may not be ideal in "live" operation. Of course you would not do that in the real-time audio thread but still.

Reply:
One more thing: if destroy() and init() how would you achieve the Pause/Continue feature ?  See #100

Reply:
I think that for Pause, you would stop sending data to the library and for Continue, you would resume sending data. So the application can just throw away audio data when paused.

Reply:
then the momentary value would not be computed any more, perhaps you still want want it ?

Reply:
Perhaps. I guess I am not familiar with the use case for pause and which measurements should be paused and why.

Reply:
Pause should just not listening to the audio coming so no further values are calculated during that time. 
While you are live you sometimes have to play something very different, e.g. for monitoring. Than just hit pause, play that other and then resume.